workspace/__lib__/xod-dev/dht/dht11-device/patch.cpp

```diff
@@ -1,6 +1,6 @@
 
 struct State {
-    uint8_t port = 0;
+    static const uint8_t port = constant_input_PORT;
 };
```

Comment:
Can this canonical implementation be reworked to the new style?

Also, the `port` should be `static constexpr`, not `static const`